Firebase/Messaging/FIRMessagingDefines.h

```diff
@@ -45,3 +45,14 @@ _Pragma("clang diagnostic pop")
     return nil; \
   } while (0)
 #endif
+
+#if defined(DEBUG)
+#define FIRMesssaging_MUST_NOT_BE_MAIN_THREAD()                                                 \
```

Comment:
Looks like this macro may be unused?

Reply:
Yeah I plan to use it in the next PR when I'm moving the db operation off main thread. I can remove from this PR if you like.

Reply:
I think it's ok to leave in, but the macro in the `#else` statement should have the same name or it'll cause unrecognized identifier errors.